go-update/src/test/java/org/reactome/release/goupdate/DuplicateReporterIT.java

```diff
@@ -28,7 +30,7 @@
 	@Before
 	public void setup() throws SQLException
 	{
-		adaptor = new MySQLAdaptor("localhost", "test_slice_64_Java_GO_Update", "root", "root", 3306);
+		adaptor = new MySQLAdaptor("localhost", "gk_central_Perl_GO_Update", "root", "root", 3306);
```

Comment:
Should this be configged?

Reply:
Yes, probably. It was a low-priority task for me, since this code never runs in production, only locally, when testing. If we ever get to the point where we run automated tests on some server then this will definitely need to be cleaned up... I'll probably get to this one later.